pkg/pool-utils/test/factories/BasePoolSplitCodeFactory.test.ts

```diff
@@ -25,8 +27,11 @@ describe('BasePoolSplitCodeFactory', function () {
 
     authorizer = await deploy('v2-vault/TimelockAuthorizer', { args: [admin.address, ZERO_ADDRESS, MONTH] });
     vault = await deploy('v2-vault/Vault', { args: [authorizer.address, WETH.address, MONTH, MONTH] });
+    protocolFeesProvider = await deploy('v2-standalone-utils/ProtocolFeePercentagesProvider', {
+      args: [vault.address, fp(1), fp(1)],
+    });
 
-    factory = await deploy('MockPoolSplitCodeFactory', { args: [vault.address] });
+    factory = await deploy('MockPoolSplitCodeFactory', { args: [vault.address, protocolFeesProvider.address] });
```

Comment:
Once more :)

Reply:
Vault.ts deploys and sets it, but if you deploy the Vault directly, you get the Vault contract itself. I did find another case in benchmarks, though